docs/dmsgpty.md

```diff
@@ -57,10 +57,11 @@ $ ./bin/dmsgpty-cli --help
 
 ```
 
-### Example usage
-
+## Example usage
 In this example, we will use the `dmsg` network where the `dmsg.Discovery` address is `http://dmsg.discovery.skywire.skycoin.com`. However, any `dmsg.Discovery` would work.
 
+### Share Files
```

Comment:
Not sure why we would call this `Share Files`.

Reply:
Can you suggest a heading?

Reply:
We can just call it `Seting up remote command execution over dmsg`

Reply:
Okay